src/rest-server/src/models/v2/group.js

```diff
@@ -21,7 +21,6 @@ const authConfig = require('@pai/config/authn');
 const secretConfig = require('@pai/config/secret');
 const adapter = require('@pai/utils/manager/group/adapter/externalUtil');
 const config = require('@pai/config/index');
-const userModel = require('@pai/models/v2/user');
```

Comment:
Is it necessary to do this change? Because you do the same operation in the function.

Reply:
Workaround for circular dependency, see http://www.ruanyifeng.com/blog/2015/11/circular-dependency.html.

User model require group model, then if we require user model in group model in turn, here will get a empty userModel.

Reply:
Create issue about this

Reply:
After checkin